source/installation-guide/wazuh-server/step-by-step.rst

```diff
@@ -77,7 +77,18 @@ Installing the Wazuh manager
 
       .. include:: /_templates/installations/wazuh/common/check_wazuh_manager.rst
 
+  #. Add the username and password that the Wazuh manager uses to connect to the indexer, use the keystore tool: 
 
+    .. code-block:: console
+
+       # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
+       # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>   
+
+    .. note:: The default <INDEXER_USERNAME> is admin, as the password is randomly generated, run the following command to get <INDEXER_PASSWORD>.
+      
+      .. code-block:: console
+
+         # tar -axf wazuh-install-files.tar wazuh-install-files/wazuh-passwords.txt -O | grep -P "\'admin\'" -A 1
```

Comment:
wazuh-install-files.tar does not exist on step-by-step installation. Step installation use admin:admin as you can see here https://documentation.wazuh.com/current/installation-guide/wazuh-indexer/step-by-step.html#testing-the-cluster-installation.

The note can clarify this admin:admin because is the step-by-step default installation credentials

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit